pulsar/trigger/subscriber/trigger.go

```diff
@@ -96,27 +121,41 @@ func (t *Trigger) Stop() error {
 }
 
 func consume(handler *Handler) {
-
 	for {
-
 		msg, err := handler.consumer.Receive(context.Background())
 		if err != nil {
-			logger.Debugf("Error while recieveing message")
-			return
+			logger.Errorf("Error while recieveing message: [%s]", err)
+			time.Sleep(5 * time.Second)
+			continue
 		}
 		out := &Output{}
-		out.Message = string(msg.Payload())
-		logger.Debugf("Message recieved [%v]", out.Message)
+		if handler.handler.Settings()["format"] != nil &&
```

Comment:
i feel that the trigger should potentially output any and during the trigger to flow mapping the dev can coerce as required... @vijaynalawade thoughts?

Reply:
@mellistibco : Since payload itself is a byte array, I think its ok to pass that as a string and let users parse it appropriately in the action. Other option would be to let users make that choice on the handler settings. e.g. Payload Type : [String, JSON, Avro].

Reply:
I did this to preserve "output as string" from the previous version of the code.  In other messaging connectors I've added a third "format" to indicate "bytes".  But the format field is not part f the OSS connector...